bin/sh-static/Makefile

```
@@ -0,0 +1,17 @@
+# $FreeBSD$
+# Build a statically linked non-CHERI shell for atf-sh tests
+# This is noticeably faster on QEMU since we don't have to emulate
+# CHERI instructions (shouldn't make much of a difference on HW though).
+.include <src.opts.mk>
+
+
+.if ${MACHINE_ABI:Mpurecap}
+NEED_COMPAT=	64
+.include <bsd.compat.mk>
+.endif
+
+PROG=	sh-static
+NO_SHARED?=	yes
+
+SRCDIR=	${.CURDIR:H}/sh
+.include	<${SRCDIR}/Makefile>
```

```suggestion
SRCDIR=	${SRCTOP}/bin/sh
.include	"${SRCDIR}/Makefile"
```

---

or just
```suggestion
.include	"${SRCTOP}/bin/sh/Makefile"
```
given it's only used once

---

bin/sh/Makefile expects SRCDIR to be set but I can obviously change that.